src/biome/text/pipeline.py

```diff
@@ -812,7 +812,11 @@ class _PreTrainedPipeline(Pipeline):
 
     def __init__(self, pretrained_path: str, **extra_args):
         self._binary = pretrained_path
-        archive = load_archive(self._binary, **extra_args)
+        archive = load_archive(
+            self._binary,
+            overrides={"dataset_reader": {"type": "interleaving", "readers": {}}},
```

Comment:
Could you please insert a little comment explaining the reason of this line?

Reply:
Done, totally forgot it!